json-form-editor/src/assets/js/schema.json

```diff
@@ -36,7 +37,11 @@
       "properties": {
         "title": {
           "type": "string",
-          "title": "Title"
+          "title": "Title",
+          "format": "textarea",
+          "options": {
+            "input_height": "20px"
```

Comment:
Would this not just be done with styles? Fine for now, but if set with styles can likely remove these `input_height` definitions.

Reply:
Yeah definitely! I think It'll get slightly tricky since these fields are causing dynamically generated markup so targeting specific inputs might be an interesting challenge, but I think it's doable perhaps based on the methodology used in the sample.

Either way, certainly worth re-evaluation once we add styling!